src/evmTest/scala/io/iohk/ethereum/vm/ThrowSpec.scala

```diff
@@ -13,7 +13,7 @@ class ThrowSpec extends FreeSpec with Matchers {
 
       val result = contract.justThrow().call()
 
-      result.error shouldBe Some(InvalidOpCode) // TODO: change to invalid jump destination
+      result.error shouldBe Some(InvalidOpCode(0xfd.toByte))
```

Comment:
funny thing is, solidity 0.4.10 is unable to compile as if it was 0.4.9. After removing `^`:
> Source file requires different compiler version (current compiler is 0.4.10+commit.59099908.Linux.g++ - note that nightly builds are considered to be strictly less than the released version
pragma solidity 0.4.9;